RELEASENOTES.md

```diff
@@ -7,6 +7,29 @@
 * Ubuntu 18.04 and 20.04 LTS
 * CentOS 7 and 8
 
+# v1.1.5 release notes
+
+This release requires [Libfabric v1.11.0](https://github.com/ofiwg/libfabric/releases/tag/v1.11.0)
+or later and supports [NCCL v2.11.4](https://github.com/NVIDIA/nccl/releases/tag/v2.11.4-1) while
+maintaining backward compatibility with older NCCL versions (up to [NCCL v2.4.8](https://github.com/NVIDIA/nccl/releases/tag/v2.4.8-1)).
+It was tested with Libfabric versions up to [Libfabric v1.14.0](https://github.com/ofiwg/libfabric/releases/tag/v1.14.0).
+
+New Features:
+* Make use of FI_EFA_FORK_SAFE environment variable to allow Libfabric to detect when `MADV_DONTFORK`
+  is not needed (#82).  This feature requires Libfabric v1.13.0 or higher.  When used with an older version
+  of Libfabric, the plugin will continue to set the RDMAV_FORK_SAFE environment variable.
+
+Bug Fixes:
+* Ensure that the buffer used for flush is page aligned and allocated with `mmap` instead of `malloc`.
+  This change is needed to correctly support `fork()` with `MADV_DONTFORK` (#77).
+
+Testing:
+The plugin has been tested with following libfabric providers using unit tests
+bundled in the source code:
+* tcp;ofi_rxm
```

Comment:
Please verify that the release has been tested with tcp and sockets provider.

Reply:
I thought we were going to test the combined plugin (including your change)?

Reply:
In any case, this change cannot affect tcp and ofi_rxm because they pay no attention to this environment variable.

Reply:
Nah, I want to keep this release independent of my changes. We have a master branch PR for this change so we would be setting fork safety for all providers and not just `efa`. Can we ensure that nothing breaks?

Reply:
True, but only the `efa` provider pays any attention.  I'll run the test though.

Reply:
Testing done